var/spack/repos/builtin/packages/py-biobb-structure-checking/package.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyBiobbStructureChecking(PythonPackage):
+    """Biobb_structure_checking provides a series of functions
+    to check the quality of a 3D structure intended to facilitate
+    the setup of a molecular dynamics simulation of protein or nucleic acids systems"""
+
+    pypi = "biobb_structure_checking/biobb_structure_checking-3.14.4.tar.gz"
+
+    maintainers("d-beltran")
+
+    # Versions
+    version("3.13.4", sha256="d819819d13c7ad219411b70b043555dcd65d5535f696a1121db562646931f445")
+
+    # Dependencies
+    depends_on("py-setuptools", type="build")
+    depends_on("python@3.8:", type=("build", "run"))
+    depends_on("py-psutil", type=("run"))
+    depends_on("py-numpy", type=("run"))
+    depends_on("py-biopython", type=("run"))
```

Comment:
```suggestion
    depends_on("py-biopython@1.79:", type=("build", "run"))
```
These should all be build/run, not just run

Reply:
I don't fully understand how spack works in this regard.
I tested locally that I was able to install and run this package by setting all these dependencies as 'run' only so I assumed they were run-only dependencies.
¿How do I know if a dependency is a build dependency as well?
Thanks for the patience!

Reply:
See https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#dependencies. Although it installs, the import tests run using `spack install --test=root` won't pass unless the dependencies are available at build-time.